apps/fishing-map/features/datasets/NewDataset.tsx

```diff
@@ -108,9 +111,11 @@ function NewDataset(): React.ReactElement {
           file.type === 'application/octet-stream' ||
           file.type === 'multipart/x-zip'
         const isGeojson =
-          (!isZip && file.type === 'application/json') || (!isZip && file.name.includes('.geojson'))
+          !isZip && (file.type === 'application/json' || file.name.includes('.geojson'))
+        const isCSV = !isZip && !isGeojson && file.type === 'text/csv'
```

Comment:
Are we sure we don't ever get `text/plain` here?

Reply:
In which use case could it happen?